packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -907,9 +913,72 @@ class SelectionOverlay {
   /// Disposes this object and release resources.
   /// {@endtemplate}
   void dispose() {
+    _androidPostDragShowTimer?.cancel();
+    _androidPostDragShowTimer = null;
     hide();
   }
 
+  bool _startHandleDragging = false;
+  bool _endHandleDragging = false;
+  bool get _draggingHandles => _startHandleDragging || _endHandleDragging;
```

Comment:
Nit: I was confused by these names at first, I thought start/end were verbs, not adjectives telling you which handle is being dragged.  Is this any better? `_isDraggingStart/EndHandle` `_isDraggingHandle`